hadoop-ozone/dist/src/test/shell/gc_opts.bats

```diff
@@ -14,14 +14,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-
 #
 # Can be executed with bats (https://github.com/bats-core/bats-core)
-# bats gc_opts.bats (FROM THE CURRENT DIRECTORY)
+# bats gc_opts.bats
 #
 
-source ../../shell/hdds/hadoop-functions.sh
+load ../../shell/hdds/hadoop-functions.sh
```

Comment:
Oh, good to now, that we have it.

Reply:
Previously it required `.bash` extension, but it's fixed in 1.2.1.